src/GeoPing.Api/Startup.cs

```diff
@@ -106,9 +107,12 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+        public void Configure
+            (IApplicationBuilder app, 
+            IHostingEnvironment env, 
+            ILoggerFactory loggerFactory)
         {
-            LoggerConfig.SetSettings();
+            LoggerConfig.SetSettings(app.ApplicationServices.GetRequiredService<IOptions<ApplicationSettings>>());
```

Comment:
Is this how it should be done? @jokerosky

Reply:
yes